presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -432,6 +449,21 @@ public boolean supportClientTransaction()
         return traceToken;
     }
 
+    @Override
+    public Optional<Tracer> getTracer()
+    {
+        return tracer;
+    }
+
+    private boolean isTracingEnabled()
+    {
+        if (this.systemProperties.getOrDefault(SystemSessionProperties.ENABLE_DISTRIBUTED_TRACING, "")
+                .equalsIgnoreCase("true")) {
+            return true;
+        }
+        return false;
```

Comment:
nit
```java
return systemProperties.getOrDefault(SystemSessionProperties.ENABLE_DISTRIBUTED_TRACING, "").equalsIgnoreCase("true");
```

Also, add a comment to explain why we can't add this to `SystemSessionProperties` class